docs/plugin_information.asciidoc

```diff
@@ -0,0 +1,16 @@
+:default_plugin: 0
+
+//////////////////////////////////////////
+GENERATED FILE DO NOT EDIT
+//////////////////////////////////////////
+* Version: %\{VERSION\}
+* Released on: %\{RELEASE_DATE\}
+* %\{CHANGELOG_URL\}[Changelog]
+
+ifeval::[{default_plugin} == 0]
+
+==== Installation
+
+For plugins not bundled by default, it is easy to install by running +bin/logstash-plugin install logstash-{type}-{plugin}+. See <<working-with-plugins>> for more details.
+
+endif::[]
```

Comment:
@whack It's not really an intermediate file. It gets generated as an intermediate step, but it's included in the docs here:

https://github.com/logstash-plugins/logstash-filter-dissect/pull/20/files#diff-d78ef6a21fe9540cd4fdb318c7516596R8